### DISCUSSION TITLE
SQLite

### DISCUSSION THREAD

**Discussion by paolodina** [romeerez/orchid-orm#31] [Ideas]
Hi, first I literally love your ORM. Seems perfect fit for my SvelteKit projects. :pray:

I agree with you about the current state of ORMs in JavaScript/TypeScript world.

Said that, I'd like to talk [about the next steps](https://github.com/romeerez/orchid-orm/issues/12#issue-1571467494), when you say:

> Meanwhile, I'll be working on the support of MariaDB/MySQL (because it's essential for the ORM to support many databases), and then SQLite.

Would you consider instead to put SQLite first? Imho, with Orchid ORM embracing Postgres we already have the best. I don't think adding MySQL would add something so great, or at least not now. Also consider that not everyone has a Postgres or MySQL instance at hand, and maybe would prefer something more handy to test the ORM (and eventually use it in the future). Or maybe they could want to use the ORM on embedded systems, etc. Or simply: I cannot ask my friends to install Postgres or MySQL to test my simple crud... :)

I guess you have good reasons for your choice, happy if you want to share them!

**Comment by romeerez** [comment]
Thank you so much, I'm very glad you liked it!

I completely agree that there is no sense in choosing MySQL (or MariaDB) when Postgres exists, while SQLite is a better choice in certain cases.

But yet MySQL is a very popular choice, so I prioritize it because more people want to use it.

Adding a second db is a really huge challenge that will take a lot of time, at the moment I have no idea how to even support docs for the two, maybe I'll have to rebuild the docs, and I'll have to update all related libraries to untie them from Postgres. But after that, adding a third database won't be such a big challenge.

For the quick testing of ORM there are cloud providers with free Postgres plans, I should point that out in the docs.
